ament_index_python/test/test_ament_index_python.py

```diff
@@ -182,3 +189,101 @@ def get_package_share_directory_test(package_name, expect_prefix):
         pass
     except Exception as exc:
         assert False, 'Expected PackageNotFoundError, got: {}'.format(type(exc))
+
+
+def test_get_resource_types():
+    set_ament_prefix_path([])
+    with pytest.raises(EnvironmentError) as excinfo:
+        get_resource_types()
+    assert excinfo.type is EnvironmentError, f'Expected EnvironmentError, got: {excinfo.type}'
```

Comment:
Since the above context manager catches only that specific exception type this assertion seems redundant and should always be satisfied.